src/Util/ClassDetails.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+use Symfony\Bundle\MakerBundle\Str;
+
+final class ClassDetails
+{
+    private $fullClassName;
+
+    public function __construct(string $fullClassName)
+    {
+        $this->fullClassName = $fullClassName;
+    }
+
+    public function getProperties():? array
```

Comment:
Make this private and move to the bottom. At least right now, we don't need this to be public.